tooling/commands/test.js

```diff
@@ -200,7 +200,10 @@ module.exports = {
           }, '');
 
           console.log(`Package ${packageName} Args ${argString}`);
-          const [cmd, ...args] = `yarn run test --silent --no-coverage-report --packages ${packageName}${argString}`.split(' ');
+          const [cmd, ...args] =
+            `yarn run test --silent --no-coverage-report --packages ${packageName}${argString}`.split(
+              ' '
+            );
```

Comment:
Looks like just a formatting change. Can we revert this?

Reply:
Removed.